lang/main.json

```diff
@@ -605,6 +605,7 @@
         "mutedRemotelyDescription": "You can always unmute when you're ready to speak. Mute back when you're done to keep noise away from the meeting.",
         "videoMutedRemotelyTitle": "Your video has been turned off by {{participantDisplayName}}",
         "videoMutedRemotelyDescription": "You can always turn it on again.",
+        "participantsJoined": "{{joinedParticipants}} people",
```

Comment:
Can we name the key something else? It means those who *want to* join, not the ones who have already joined.